### DISCUSSION TITLE
Can i debug yourls locally ? using visual studio code ?

### DISCUSSION THREAD

**Discussion by meiry** [YOURLS/YOURLS#2881] [General]
Hello all
As the title say, can i set up yourls on my local pc using nginx + php and debug it in Visual Studio Code as i do with other PHP apps ? 
if yes how does it will handle the domain?
Is there any guide?  ( not how to setup php/visualcode but just the YOURLS section )
Thanks

**Comment by ozh** [comment]
Well YOURLS is just a PHP app so yes you can run it locally "as you do with other PHP apps" ... To be honest I don't get the question :)

**Comment by minecraftchest1** [comment]
There are some php live preview extensions for visual studio code. As for the question about how it will handle the domain, I don't know what you mean.
